src/formpack/pack.py

```diff
@@ -202,11 +202,41 @@ def _combine_field_choices(old_field, new_field):
             old_choice = old_field.choice
             new_choice = new_field.choice
         except AttributeError:
-            return
+            return new_field
```

Comment:
The docstring should be changed to match this.

Reply:
done!